org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandler.java

```diff
@@ -95,11 +103,24 @@
 		return $;
 	}
 
-	private Command getCommandFromProposal(CUCorrectionProposal proposal) throws CoreException {
+	private Either<Command, CodeAction> getCodeActionFromProposal(CUCorrectionProposal proposal, CodeActionContext context) throws CoreException {
 		String name = proposal.getName();
 		ICompilationUnit unit = proposal.getCompilationUnit();
+		Command command = new Command(name, COMMAND_ID_APPLY_EDIT, Arrays.asList(convertChangeToWorkspaceEdit(unit, proposal.getChange())));
 
-		return new Command(name, COMMAND_ID_APPLY_EDIT, Arrays.asList(convertChangeToWorkspaceEdit(unit, proposal.getChange())));
+		String kind = proposal.getKind();
+		if (preferenceManager.getClientPreferences().isSupportedCodeActionKind(kind)) {
+			CodeAction codeAction = new CodeAction(name);
+			if (CodeActionKind.QuickFix.equals(kind)) {
+				kind = REFACTOR_QUICKFIX;
```

Comment:
No. Quick fixes are no longer quick fixes that way, so no actions available from the problems view in vscode